### ISSUE TITLE
Error when installing mingw-w64-x86_64-adwaita-icon-theme

### ISSUE THREAD

**Issue by PHdrr** [msys2/MINGW-packages#16142]
```
( 7/10) installing mingw-w64-x86_64-adwaita-icon-theme                                                         [#################################################################] 100%
error: command (/usr/bin/bash /usr/bin/bash -c . /tmp/alpm_mDntxS/.INSTALL; post_install 43-1 ) failed to execute correctly
```

**Comment by Biswa96** [comment]
What is the version of that package? Have you try to re-run `pacman -Syyu` command?

**Comment by PHdrr** [comment]
> What is the version of that package? Have you try to re-run `pacman -Syyu` command?

It seems this error is always there. I have done many new installations of MSYS2 on many PCs and it will always error on that particular package when installing `mingw-w64-x86_64-gtk3`.

p/s: My MSYS2 is always up to date. I always run `pacman -Syu` before starting to install packages.

**Comment by lazka** [comment]
In theory this was fixed today -> #16134

**Comment by Biswa96** [comment]
From the log, it seems that you are using 43-1 version but the latest one is 43-2. See https://packages.msys2.org/base/mingw-w64-adwaita-icon-theme

**Comment by PHdrr** [comment]
> From the log, it seems that you are using 43-1 version but the latest one is 43-2. See https://packages.msys2.org/base/mingw-w64-adwaita-icon-theme

The log is from yesterday. I just did a `pacman -Syu` but didn't notice if this package was also updated.

**Comment by PHdrr** [comment]
> In theory this was fixed today -> #16134

Thank you. So this bug was already fixed. I will close the issue.
